File: Patterns/7. Pyramid Patterns/61. Pyramid Stars/ps.java
import java.util.*;
public class ps{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            // print spaces
            for(int j=1; j<=n-i; j++){
                System.out.print("  ");
            }
            // print stars
            for(int j=1; j<=2*i-1; j++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

/*
Output

        * 
      * * * 
    * * * * * 
  * * * * * * * 
* * * * * * * * * 

Another Method (Matrix form) using if-else:

    System.out.print("Enter a Number : ");
    int p = sc.nextInt();

    for(int i=1; i<=p; i++){
        for(int j=1; j<=2*p-1; j++){
            if(j>=p-i+1 && j<=p+i-1){
                System.out.print("* ");
            }else{
                System.out.print("  ");
            }
        }
        System.out.println();
    }

*/

File: Patterns/07. Pyramid Patterns/61. Pyramid Stars/ps.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    # print spaces
    for j in range(1,n-i+1):
        print(end='  ')
    # print stars
    for j in range(1,2*i):
        print("*",end=" ")
    print()

'''
Output:

        * 
      * * * 
    * * * * * 
  * * * * * * * 
* * * * * * * * * 

Another Method using if-else

p = int(input("Enter a Number : "))

for i in range(1,p+1):
    for j in range(1,2*p):
        if(p-i+1 <= j <= p+i-1):
            print("*",end=" ")
        else:
            print(end='  ')
    print()

'''